soc/v2/raspberrypi/rp2/Kconfig

```diff
@@ -4,14 +4,9 @@
 # Copyright (c) 2021 Yonatan Schachter
 # SPDX-License-Identifier: Apache-2.0
 
-choice
-	prompt "RP2xxx MCU Selection"
-	depends on SOC_SERIES_RP2XXX
-
 config SOC_RP2040
-	bool "Raspberry Pi RP2040"
-
-endchoice
+	bool
+	select SOC_SERIES_RP2XXX
```

Comment:
The SoC definition must be in a `Kconfig.soc` so that it can be sourced elsewhere, such as sysbuild.